packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -418,7 +591,7 @@ class ResidentWebRunner extends ResidentRunner {
   @override
   Future<void> debugDumpRenderTree() async {
     try {
-      await _vmService.callServiceExtension(
+      await _vmService?.callServiceExtension(
```

Comment:
Does this happen when the user uses presses one of the shortcut keys before the VM service is initialized? If so, maybe print something along those lines to the console so we don't just swallow keystrokes?

Reply:
There is no vmservice on yet on the incremental mode, that is the only reason for the `?`